tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -11,6 +12,15 @@ using namespace shogun;
 using namespace linalg;
 using namespace Eigen;
 
+template <typename Types>
+struct TypesGoogleTestWrapper;
+
+template <template<typename...> class TypesT, typename... Args>
+struct TypesGoogleTestWrapper<TypesT<Args...>>
+{
+    typedef ::testing::Types<Args...> type;
+};
```

Comment:
@karlnapf would it make sense to put this in a header that can be accessed by all tests? Together with the ::testing::Types typedefs below?

Reply:
that would be great... if you do that also make sure to use it from at least one other test, and then maybe even create an entrance task for people to make use of this in all shogun tests ....